Client/src/Components/Search/SearchViewModal.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Button, Grid, Modal, Paper } from "@material-ui/core"
+import React, { useState } from 'react'
+
+import { IDatasetModel } from "../../Models/Datasets/IDatasetModel"
+import SearchView from "./SearchView"
+import { classStyles } from "../../appTheme"
+
+interface IProps {
+  onDatasetsSelected: (foundDatasets: IDatasetModel[]) => void
+}
+
+export const SearchViewModal = (props: IProps) => {
+  const { onDatasetsSelected } = { ...props }
+  const [open, setOpen] = useState(false)
+
+  const handleDatasetsSelected = (foundDatasets: IDatasetModel[]) => {
+    onDatasetsSelected(foundDatasets)
+    setOpen(false)
+  }
+
+  return (
+    <>
+      <Grid item>
+        <Button id="add-dataset" onClick={() => setOpen(true)} color="primary" variant="contained">Add dataset To Graph</Button>
+      </Grid>
+      <Modal
```

Comment:
Please put back the cancel icon button that was removed. It is still in dev and should not be removed since it was a requirement from the stakeholder